presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTableStatistics.java

```diff
@@ -14,6 +14,7 @@
 package io.prestosql.tests.hive;
 
 import com.google.common.collect.ImmutableList;
+import io.prestosql.sql.planner.planprinter.IoPlanPrinter;
```

Comment:
by general we tend to not to use presto main code in product tests. However this usage seems reasonable.